TLM/TLM/Constants.cs

```diff
@@ -8,6 +8,7 @@
 namespace TrafficManager {
 	public static class Constants {
 		public static readonly bool[] ALL_BOOL = new bool[] { false, true };
+		public const float BYTE_TO_FLOAT_OFFSET_CONVERSION_FACTOR = 0.003921569f;
```

Comment:
Magical constant is terrible. One can easily see that it is 1.0f / 255f

Reply:
Yes, but it is still better than dividing 1f/255f each time

Reply:
It's a static class, so surely it will only do the actual calculation at compile time?

Reply:
Compilers know exactly when a part of expression can be calculated in the compile time, you don't need to pre-calculate for them, it is important for whoever will read it later, to understand the meaning of this constant. I will change it later when i visit this file and find it, don't bother fixing it right now.

Reply:
tbh, even if it's changed to `1f/255f` (which we should do IMO), I still wouldn't have a clue what it does or how it works -- is "byte to float offset conversion factor" something that's commonly understood?

Reply:
it's simple: for each byte (0-255) you will get fraction of 1f
- 0(byte) == 0f, 
- 255(byte) == 1f

Reply:
Yup, that would be helpful, even if comment was just a URL to a stackoverflow (or whatever) explaining what's going on.